solr/bin/solr

```diff
@@ -1304,7 +1191,7 @@ function start_solr() {
     # '+CrashOnOutOfMemoryError' ensures that Solr crashes whenever
     # OOME is thrown. Program operation after OOME is unpredictable.
     "-XX:+CrashOnOutOfMemoryError" "-XX:ErrorFile=${SOLR_LOGS_DIR}/jvm_crash_%p.log" \
-    "-Djetty.home=$SOLR_SERVER_DIR" "-Dsolr.solr.home=$SOLR_HOME" "-Dsolr.install.dir=$SOLR_TIP" "-Dsolr.install.symDir=$SOLR_TIP_SYM" \
+    "-Dsolr.jetty.home=$SOLR_SERVER_DIR" "-Dsolr.solr.home=$SOLR_HOME" "-Dsolr.install.dir=$SOLR_TIP" "-Dsolr.install.symDir=$SOLR_TIP_SYM" \
```

Comment:
Any chance that this is an opportunity to rethink all these various flavours of path?   Could we just have a single value, and derive everything esle?  Do we really need control over these?   Also, if we kept them as env variables, couldn't you just read them IN as part of the java code?     I find all these various settings quite confusing to keep straight.    Last thought in my long run-opn thought...   I would love to have the `production.properties` or `production.yml` type file define all these, and just pass that into `solr-start.jar`.

Reply:
Sure, probably lots of cleanup to be done there. But in this POC my goal is to prove that it is not a huge change to do this thing, and the startup command line is almost the same. I just normalized some sys props here.